nftables.go

```diff
@@ -269,3 +318,44 @@ func (n *nft) ShutDown() error {
 	}
 	return nil
 }
+
+// Utilites from https://github.com/IBM/netaddr/blob/master/net_utils.go
+
+// NewIP returns a new IP with the given size. The size must be 4 for IPv4 and
+// 16 for IPv6.
+func NewIP(size int) net.IP {
+	if size == 4 {
+		return net.ParseIP("0.0.0.0").To4()
+	}
+	if size == 16 {
+		return net.ParseIP("::")
+	}
+	panic("Bad value for size")
```

Comment:
don't panic here please, return error instead

Reply:
This code is ripped from https://github.com/IBM/netaddr/blob/master/net_utils.go , It won't ever to the panic since this we don't directly use it. But I'll make the changes, panic is scary